dev/com.ibm.ws.security.audit.source/src/com/ibm/ws/security/audit/encryption/AuditCrypto.java

```diff
@@ -1658,6 +1704,9 @@ static final byte[][] rsaKey(int len, boolean crt, boolean f4) {
         } catch (java.security.NoSuchAlgorithmException e) {
             Tr.error(tc, "security.ltpa.noalgorithm", new Object[] { e });
             com.ibm.ws.ffdc.FFDCFilter.processException(e, "com.ibm.ws.security.audit.AuditCrypto", "1796");
+        } catch (NoSuchProviderException e1) {
+            Tr.error(tc, "No Such Provider");
```

Comment:
```suggestion
            if (tc.isDebugEnabled()) {
                Tr.debug(tc, "No Such Provider");
            }
```
Is FFDC needed like what is done in the previous catch?